libbeat/common/errors.go

```diff
@@ -15,7 +15,7 @@
 // specific language governing permissions and limitations
 // under the License.
 
-package input
+package common
```

Comment:
Why moving this to common? I think it is ok if we have to explicitly import this error for the input package.

Reply:
I thought it won't be a good idea to import a `filebeat` package (input) within a `libbeat` package. To my mind `libbeat` is superior from various beats' codebases. Nevertheless I gave it a try and it seems that it leads to a circular import indeed 🙂 :
```
import cycle not allowed
package github.com/elastic/beats/v7/filebeat
	imports github.com/elastic/beats/v7/filebeat/cmd
	imports github.com/elastic/beats/v7/filebeat/beater
	imports github.com/elastic/beats/v7/filebeat/autodiscover/builder/hints
	imports github.com/elastic/beats/v7/filebeat/fileset
	imports github.com/elastic/beats/v7/libbeat/cfgfile
	imports github.com/elastic/beats/v7/filebeat/input
	imports github.com/elastic/beats/v7/filebeat/channel
	imports github.com/elastic/beats/v7/libbeat/cfgfile
```

Reply:
Oh, I see. Ok, let's go with this by now.